Wikipedia/Code/WMFCVLSection.h

```diff
@@ -0,0 +1,40 @@
+#import <Foundation/Foundation.h>
+
+@class WMFCVLColumn;
+@class WMFCVLAttributes;
+@class WMFCVLInvalidationContext;
+
+/*!
+ @class        WMFCVLSection
+ @abstract     A WMFCVLSection represents a section within a column of a WMFColumnarCollectionViewLayout. It handles adjusting the size and offset of items within the section.
+ @discussion   ...
+ */
+@interface WMFCVLSection : NSObject
+
+@property (nonatomic, readonly) NSInteger index;
+@property (nonatomic) CGRect frame;
+@property (nonatomic, weak, nullable) WMFCVLColumn *column;
```

Comment:
it’s for this lookup:

```
- (void)updateWithInvalidationContext:(nonnull WMFCVLInvalidationContext *)context delegate:(id <WMFColumnarCollectionViewLayoutDelegate>)delegate collectionView:(UICollectionView *)collectionView {
    if (context.boundsDidChange) {
        [self resetSections];
        [self layoutForBoundsSize:context.newBounds.size withDelegate:delegate collectionView:collectionView invalidationContext:context];
    } else if (context.originalLayoutAttributes && context.preferredLayoutAttributes) {
        UICollectionViewLayoutAttributes *originalAttributes = context.originalLayoutAttributes;
        UICollectionViewLayoutAttributes *preferredAttributes = context.preferredLayoutAttributes;
        NSIndexPath *indexPath = originalAttributes.indexPath;
        
        WMFCVLSection *invalidatedSection = self.sections[indexPath.section];
        WMFCVLColumn *invalidatedColumn = invalidatedSection.column;
```

There might be a better way to do it. You could ask each column if that section index is in it’s index set.